src/tests/functional/inference_engine/transformations/simplify_second_input_of_reshape_test.cpp

```diff
@@ -18,12 +18,13 @@
 using namespace testing;
 using namespace ngraph;
 
+namespace {
```

Comment:
why is this added?

Reply:
as explained in description of commit 4, it fixes odr violation